theme/index.hbs

```diff
@@ -188,6 +188,9 @@
                           <li role="none"><button role="menuitem" class="theme">
                               <a id="zh-TW">Chinese Traditional (漢語)</a>
                           </button></li>
+                          <li role="none"><button role="menuitem" class="theme">
+                              <a id="ko">Japanese (日本語)</a>
```

Comment:
This seems to be linking to the Korean translation instead.

Reply:
Thanks for the catch, I'm sending out a fix right away.